main/src/ca/uwaterloo/flix/language/ast/shared/SecurityContext.scala

```diff
@@ -24,6 +24,11 @@ sealed trait SecurityContext
 
 object SecurityContext {
 
+  /**
+    * A security context where no unsafe features are permitted along with the IO effect.
+    */
+  case object Paranoid extends SecurityContext
```

Comment:
Whats the order here? Alphabetical? We could also use the order you list them in below, i.e. following the lattice.